lib/services/elasticsearch.js

```diff
@@ -1480,19 +1523,21 @@ class ElasticSearch extends Service {
             successes.push({
               _id: item._id,
               _source: documents[i]._source,
-              _status: item.status,
+              status: item.status,
               _version: item._version,
-              result: item.result
+              created: item.created,
+              result: item.result,
+              get: item.get // user by mUpdate to get the full document body
             });
           }
         }
 
         return {
-          result: successes,
+          items: successes, // @todo rename items to documents
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)